src/openapi/models/model_409_error.rs

```diff
@@ -19,13 +19,19 @@ pub struct Model409Error {
     pub error: String,
 }
 
-///
+/// 
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
 pub enum Code {
     #[serde(rename = "user_already_admin")]
-    AlreadyAdmin,
+    UserAlreadyAdmin,
     #[serde(rename = "user_already_organization_member")]
-    AlreadyOrganizationMember,
+    UserAlreadyOrganizationMember,
     #[serde(rename = "user_has_no_passkeys")]
-    HasNoPasskeys,
+    UserHasNoPasskeys,
```

Comment:
The renames of enum values here by the codegen could be considered a breaking change.

But since the crate is still in beta (<v1.0) and this type is unlikely to be directly in use by a consumer, the PR title is just labeled as a feature. If the preferred pattern would be to call it out as breaking instead, the only additional consideration would be to likely try to roll this PR up with other breaking changes before releasing.

Reply:
I think we can call it a breaking change via `feat!:` in this case since we're planning on only doing a major release.

i think i'd be fine with `feat` as well, but given our plan for flex sdks i don't think it would make a difference.